### PR TITLE
Examine and resolve API breaks for in-flight SARIF v2. WIP

### PR DESCRIPTION
**Pull request by michaelcfanning**
@lgolding, once 2019-01-24 ships to NuGet, we can update the project references in this branch and ship. i did this from curiosity as much as from a desire to get a head start on that. :)

### CODE REVIEW

**Comment by lgolding** [comment]
This would be a good place to use the new GitHub ["draft pull request"](https://github.blog/2019-02-14-introducing-draft-pull-requests/) feature.

File: src/Json.Schema.Validation/ExtensionMethods.cs [hunk 1]
```diff
@@ -84,26 +84,4 @@ public static Result ToSarifResult(this JsonSyntaxException ex)
             };
         }
     }
-
-    public static class RuleConfigurationDefaultLevelExtensions
```
Comment (lgolding) [inline src/Json.Schema.Validation/ExtensionMethods.cs]: Nice.

**Comment by lgolding** [comment]
This all looks very nice. Is that all the changes, and now we're just waiting to publish the SARIF SDK packages so you can fix up the project files? If so, let me know and I'll mark the PR "Approved."

**Comment by michaelcfanning** [comment]
Yes, this is all just waiting on the NuGet publish of TC32 to close.

---
In reply to: [464491751](https://github.com/Microsoft/jschema/pull/99#issuecomment-464491751) [](ancestors = 464491751)

**Comment by michaelcfanning** [comment]
@lgolding updated to nuget-delivered packages and so merging.

**Pull request merged**